plaso/parsers/winreg_plugins/amcache.py

```diff
@@ -235,6 +239,8 @@ def _ParseApplicationSubKey(self, parser_mediator, application_sub_key):
       event_data.link_time = self._ParseDateStringValue(
           parser_mediator, application_sub_key.path, link_date_value)
 
+    event_data.last_written_time = application_sub_key.last_written_time
```

Comment:
at the moment this attribute is used for "AEINV_WER modification date and time" also see https://github.com/libyal/dtformats/blob/main/documentation/AMCache%20file%20(AMCache.hve)%20format.asciidoc#611-file-sub-key